src/app/header/header.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-toolbar class="head-panel" color="primary" *ngIf="currentUser">
+  <div>
+  <img class="header-logo" src="../../assets/img/qp_logo.png" alt="logo" width="40px" hight="40px">
+</div>
+<span class="toolbar-header">Quick Pack</span>
```

Comment:
`<span>` is an inline tag designed for textual content, my suggestion is to change it to `section` or `div`